### PR TITLE
Id converted to class

### LINKED ISSUE #4619: siteSearch is id but being continuously used in the same page
Problem : Making code refined, 
There are many files where i found id siteSearch, I want to report a file which have more that one instance of it

/openlibrary/templates/stats/readinglog.html

so I want to know if its better to change the id to class or better to create a new class for the readinglog.html file

There are many files with id siteSearch and many files with class siteSearch 

I think it will be better to change the id to class with some other name 

 
### Stakeholders

@jdlrobson

### PR DESCRIPTION
**Pull request by ArunTeltia**
Closes #4619

@jdlrobson

### CODE REVIEW

File: openlibrary/macros/SearchResults.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 $ _ = i18n.get_namespace('/search')
 
-<ul id="siteSearch">
+<ul id="list-books">
```
Comment (jdlrobson) [inline openlibrary/macros/SearchResults.html:5]: should be a class not an ID.

File: openlibrary/templates/work_search.html [hunk 2]
```diff
@@ -212,7 +212,7 @@
         <div class="shift">$ungettext('1 hit', '%(count)s hits', num_found, count=commify(num_found))</div>
         <div class="resultsContainer search-results-container">
         <div id="searchResults">
-          <ul class="list-books" id="siteSearch">
+          <ul class="list-books" >
```
Comment (jdlrobson) [inline openlibrary/templates/work_search.html:215]: nitpick: space not necessary here.

File: static/css/legacy.less [hunk 3]
```diff
@@ -1662,8 +1662,8 @@ span.actions {
 // openlibrary/templates/account/books.html
 // openlibrary/macros/SearchResults.html
 // openlibrary/macros/FulltextResults.html
-// why not form#siteSearch ?
-ul#siteSearch {
+// why not form.list-books ?
```
Comment (jdlrobson) [inline static/css/legacy.less:1665]: this comment no longer makes sense and can be removed.
We now understand there is a form#siteSearch and a ul#siteSearch but that is no longer the case.

**Review by jdlrobson — CHANGES_REQUESTED** [review]

File: openlibrary/templates/work_search.html (hunk 2 above)
Comment (ArunTeltia) [reply openlibrary/templates/work_search.html]: removed will also remember this in future 🚀

**Review by jdlrobson — APPROVED** [review]
Code looks good. Just needs to be verified locally. I'll aim to do this by the end of the week but other mergers should feel free!

**Review by jdlrobson — APPROVED** [review]
Not seeing any visual problems here on http://localhost:8080/search?q=title%3A+%22Sewing%22&mode=ebooks&has_fulltext=true and http://localhost:8080/people/openlibrary/books/want-to-read

It does seem like a lot of these CSS rules are not applying but we can consolidate them later.

**Pull request merged**